process.cabal

```diff
@@ -1,14 +1,14 @@
+cabal-version: 2.4
 name:          process
-version:       1.6.18.0
```

Comment:
If I'm not mistaken this isn't *necessarily* a major version bump under the PVP since things have only been added. I'm rather of mixed feelings about this; it seems wrong for such a significant addition to only be a minor bump but on the other hand major bumps are quite a headache for everyone involved.

Reply:
It's common for core libraries to add lots of new functionality without bumping a major version, `filepath-1.4.100.0` is a prominent example of it. If there are no breaking changes, I'd suggest `1.6.19.0` or maybe `1.6.100.0` if it suits you better. Propagating major bumps through the ecosystem is way too expensive.

Reply:
Thanks for your input; I reverted back to using a minor version bump.